modules/installation-osp-creating-image.adoc

```diff
@@ -0,0 +1,49 @@
+//Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-user.adoc
+
+[id="installation-osp-creating-image_{context}"]
+= Creating the {op-system-first} image
+
+The {product-title} installation program requires that a {op-system-first} image be present in the {rh-openstack-first} cluster. Retrieve the latest {op-system} image, then upload it using the {rh-openstack} CLI.
+
+.Prerequisites
+
+* The {rh-openstack} CLI is installed.
+
+.Procedure
+
+. Log in to the Red Hat customer portal's https://access.redhat.com/downloads/content/290[Product Downloads page].
+
+. Under *Version*, select version 4.4.0 for RHEL 8.
+
+. Download the _Red Hat Enterprise Linux CoreOS - OpenStack Image (QCOW)_.
++
+[IMPORTANT]
+====
+The {op-system} images might not change with every release of {product-title}.
+You must download images with the highest version that is less than or equal to
+the {product-title} version that you install. Use the image versions that match
+your {product-title} version if they are available.
+====
+
+. Decompress the image.
++
+[NOTE]
+====
+You must decompress the OpenStack image before the cluster can use it.
+====
```

Comment:
The name of the downloaded file might not contain a compression extension (.gz, or .tgz). To find out if it is compressed and what type of compression,  use the command: 
file <name_of_downloaded_file>
The output of the will tell you if it is an archive and what type.